CMSgov/mint-app
# [EASI-4023] New Discussion Reply Notification

# EASI-4023, EASI-3944


## Changes and Description

- Made adjustments to `IndividualNotification.tsx` to show content when in mobile (matches figma now)
- Add new type guard to `_utils` file
- Add `NewDiscussionRepliedActivityMeta` fragment to `GetNotifications.ts` gql query
- Updated Notification Cypress test



## How to test this change

1. Create a new discussion
2. Reply to that discussion
3. Check to see notification will pop up
4. Verify things match figma
5. Verify tests are workign



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
